tailor_distro/manage/pin.py

```diff
@@ -90,4 +90,4 @@ def execute(self, rosdistro_repo, repositories):
             # TODO(pbovbel) store name of pinner?
             data.status_description = f"Pinned {age} commits behind {source_branch} on {datetime.datetime.now()}"
 
-        self.rosdistro_repo.write_internal_distro('Pinning {}'.format(', '.join(actions)))
+            self.rosdistro_repo.write_internal_distro('Pinning {}'.format(', '.join(actions)))
```

Comment:
this change makes it much easier to pin packages if something fails - should've PRed it ages ago, sorry for the unrelated